WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Preview/ProductDetailPreviewView.swift

```diff
@@ -316,14 +438,19 @@ fileprivate extension ProductDetailPreviewView {
         )
         static let productNameAndDescription = NSLocalizedString(
             "productDetailPreviewView.productNameAndDescription",
-            value: "Name & Description",
-            comment: "Title of the name and description fields on the add product with AI Preview screen."
+            value: "Name, Summary & Description",
```

Comment:
Should we change the key to this string as well? I still see the old text "Name & Description", I suppose this was due to the old translation being used.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/13255/commits/e1ef76bc84f3ac53e057391b15c55d04f53996c9